### ISSUE TITLE
Audio not played if not already downloaded

### ISSUE THREAD

**Issue by Navidar** [nextcloud/android#9216]
### Steps to reproduce
1. 
2. 
3. 

### Expected behaviour
- Tell us what should happen

### Actual behaviour
- Tell us what happens

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by AlvaroBrey** [comment]
Please fill the information template, otherwise we can't help.

**Comment by Navidar** [comment]
I try to explain what I observed.
My expectation was that the audio app starts playing the audio file, regardless of the file of being already downloaded or not yet. It would download and play it. If not yet downloaded, the app doesn't do anything.

another case: While playing a downloaded file, if you pause it, switch to another app and come back, it doesn't resume with the error message: "the built-in media player is unable to play the media file."  If you then press back botton of the andoird device, the app crashes with the log file I submitted previousely.

Hope it helps. Thanks!
